active_learning/our_labeler.py

```diff
@@ -18,10 +44,21 @@ def __init__(self, **kwargs):
         pusher.connection.bind('pusher:connection_established', self.connect_handler)
         pusher.connect()
 
-    # Feature is a dictionary with url and id as keys
     def label(self, feature):
+        '''
+        Return the label produced by the oracle
+
+        Parameters
+        ----------
+        self : self reference
```

Comment:
remove this, as we don't actually pass a self reference when calling this function